src/components/generics/date-picker/date-picker.tsx

```diff
@@ -30,7 +30,7 @@ const GenericDatePicker: React.FC<Props> = ({
 
   const { locale } = useAppSelector(selectLocale);
 
-  const [ todayDate /* setTodayDate */ ] = React.useState(new Date());
+  const [ todayDate ] = React.useState(new Date());
```

Comment:
But does this actually need to be state if it is not getting updated at any point or could it just be a const?